index.js

```diff
@@ -173,6 +179,7 @@ function buildFetch(behavior) {
         json: options.json,
         method: options.method,
         followRedirect: followRedirect,
+        rejectUnauthorized: rejectUnauthorized,
```

Comment:
Do we need to know if we have set `rejectUnauthorized`in the `inRequestInit` hook?